wrapdocker

```diff
@@ -7,10 +7,10 @@ dmsetup mknodes
 CGROUP=/sys/fs/cgroup
 : {LOG:=stdio}
 
-[ -d $CGROUP ] || 
+[ -d $CGROUP ] ||
```

Comment:
Excuse me for this line change. Atom removed a space. I can refactor the commit to include only the important lines.